CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Documentation
 - Improve UX for "knobs" form on component examples @levithomason ([#20](https://github.com/stardust-ui/react/pull/20))
 
+### Features
+- Add Icon `xSpacing` prop @Bugaa92 ([#22](https://github.com/stardust-ui/react/pull/22))
+- Add Button `icon` prop @Bugaa92 ([#13](https://github.com/stardust-ui/react/pull/13))
+
```

Comment:
As far as I can see, this PR is also adding the `truncated` prop for the Text component. Please add that in the CHANGELOG as well.

Reply:
done, thx